test/e2e_flags.go

```diff
@@ -73,9 +82,47 @@ func initializeFlags() *EnvironmentFlags {
 
 	flag.StringVar(&f.Tag, "tag", "latest", "Provide the version tag for the test images.")
 
+	klog.InitFlags(nil)
+	flag.Set("v", "2") // A normal level of logging for testing
```

Comment:
Why 2? Why not make this a constant

`DefaultLogLevel = 2`

Then you can use this in the unit test below, test that default is getting set properly, and be able to change the default without updating the test.

Reply:
`defaultLogLevel`, I guess?

Reply:
Adding comment why "2", and using your suggestion

Reply:
Oh, good point, no reason to export.